.gitlab/functional_test/regression_detector.yml

```diff
@@ -106,6 +100,8 @@ single-machine-performance-regression_detector:
       --comparison-image ${COMPARISON_IMAGE}
       --baseline-sha ${BASELINE_SHA}
       --comparison-sha ${CI_COMMIT_SHA}
+      --replicas 13
+      --total-samples 90
```

Comment:
What's the goal we're trying to achieve with these two settings? The second would mean we'd run the Agent for 90 seconds post-warmup. Several quality gates failures for the `quality_gates_idle` experiment occur towards the end of replicates that run for 600 seconds post-warmup.

Reply:
oh shoot, these were for testing only, thanks, will remove now.